src/views/TechnicalAssistance/RequestForm/Documents.tsx

```diff
@@ -1,29 +1,501 @@
+    GetTrbRequestDocuments,
+    GetTrbRequestDocumentsVariables
+  >(GetTrbRequestDocumentsQuery, {
+    variables: { id: request.id }
+  });
+
+  // console.log('loading', loading, 'error', error);
+  // console.log('data', data);
+
+  const documents = data?.trbRequest.documents || [];
+  // console.log('documents', documents);
+
+  // Documents can be created from the upload form
+  const [createDocument /* , createResult */] = useMutation<
+    CreateTrbRequestDocument,
+    CreateTrbRequestDocumentVariables
+  >(CreateTrbRequestDocumentQuery);
+
+  const [isUploadError, setIsUploadError] = useState(false);
+
+  // Documents can be deleted from the table
+  const [deleteDocument /* , deleteResult */] = useMutation<
+    DeleteTrbRequestDocument,
+    DeleteTrbRequestDocumentVariables
+  >(DeleteTrbRequestDocumentQuery);
+
+  const columns = useMemo<Column<TrbRequestDocuments>[]>(() => {
+    return [
+      {
+        Header: t<string>('documents.table.header.fileName'),
+        accessor: 'fileName'
+      },
+      {
+        Header: t<string>('documents.table.header.documentType'),
+        accessor: ({ documentType: { commonType, otherTypeDescription } }) => {
+          if (commonType === TRBDocumentCommonType.OTHER) {
+            return otherTypeDescription || '';
+          }
+          return t(`documents.upload.type.${commonType}`);
         }
-      }}
-      next={{
-        onClick: e => {
-          history.push(stepUrl.next);
+      },
+      {
+        Header: t<string>('documents.table.header.uploadDate'),
+        accessor: 'uploadedAt',
+        Cell: ({ value }) => DateTime.fromISO(value).toFormat('MM/dd/yyyy')
+      },
+      {
+        Header: t<string>('documents.table.header.actions'),
+        accessor: ({ status }) => {
+          // Repurpose the accessor to use `status` for sorting order
+          if (status === TRBRequestDocumentStatus.PENDING) return 1;
+          if (status === TRBRequestDocumentStatus.AVAILABLE) return 2;
+          if (status === TRBRequestDocumentStatus.UNAVAILABLE) return 3;
+          return 4;
         },
-        text: t('documents.continueWithoutAdding'),
-        outline: true
-      }}
-      taskListUrl={taskListUrl}
-    />
+        Cell: ({ row }: CellProps<TrbRequestDocuments, string>) => {
+          // Show the upload status
+          // Virus scanning
+          if (row.original.status === TRBRequestDocumentStatus.PENDING)
+            return (
+              <em data-testurl={row.original.url}>
+                {t('documents.table.virusScan')}
+              </em>
+            );
+          // View or Remove
+          if (row.original.status === TRBRequestDocumentStatus.AVAILABLE)
+            // Show some file actions once it's available
+            return (
+              <>
+                {/* View document */}
+                <Link target="_blank" href={row.original.url}>
+                  {t('documents.table.view')}
+                </Link>
```

Comment:
Do you know if this is intended to not function locally?  This url is supposed to open the document in a new tab, but  throwing error that the docker container cannot be reached.

Reply:
![Screenshot 2022-12-30 at 10 42 05 AM](https://user-images.githubusercontent.com/95709965/210087778-5c4e88a0-b5d9-41ca-82ad-4fdf09c405c8.png)

![Screenshot 2022-12-30 at 10 42 25 AM](https://user-images.githubusercontent.com/95709965/210087799-3ac5387b-e2b5-4ed0-8180-0e5c16f203d8.png)

Reply:
It could be that host.docker.internal isn't set up on your machine? Check your host route to minio.
See some details here https://github.com/CMSgov/easi-app/pull/1819

Reply:
Looks like Clay had a pr related to this [here](https://github.com/CMSgov/easi-app/pull/1830)